v03_pipeline/lib/reference_data/config.py

```diff
@@ -136,14 +136,15 @@ def custom_gnomad_select_v3(ht):
     selects['AC'] = ht.freq[global_idx].AC
     selects['Hom'] = ht.freq[global_idx].homozygote_count
 
+    grpmax_af = ht.grpmax['gnomad'].AF if hasattr(ht.grpmax, 'gnomad') else ht.grpmax.AF
```

Comment:
when I look on the sites table, it doesn't look like ht.grpmax.AF exists . This is what I see when I type ht.grpmax.describe() seen below. Is this being added somewhere in this script? 

```
--------------------------------------------------------
Type:
        struct {
        gnomad: struct {
            AC: int32, 
            AF: float64, 
            AN: int32, 
            homozygote_count: int64, 
            gen_anc: str
        }, 
        non_ukb: struct {
            AC: int32, 
            AF: float64, 
            AN: int32, 
            homozygote_count: int64, 
            gen_anc: str
        }
    }
```

Reply:
```suggestion
    grpmax_af = ht.grpmax.gnomad.AF if hasattr(ht.grpmax, 'gnomad') else ht.grpmax.AF
```
is there a reason this code uses ['arg'] instead of .arg. here? Very stupid comment and I know it's consistent, but wanted to ask

Reply:
The schema for the genomes and exomes is different here:

```
In [8]: hl.read_table('gs://gcp-public-data--gnomad/release/4.0/ht/genomes/gnomad.genomes.v4.0.sites.ht').grpmax.AF
Out[8]: <Float64Expression of type float64>

In [9]: hl.read_table('gs://gcp-public-data--gnomad/release/4.0/ht/exomes/gnomad.exomes.v4.0.sites.ht').grpmax.gnomad.AF
Out[9]: <Float64Expression of type float64>
```

Reply:
Yeah re the attribute vs dict access, it's literally just to stay the same with [this line](https://github.com/broadinstitute/seqr-loading-pipelines/pull/647/files/825d38aa269bfa658ecc4971e147bdc4b64103aa#diff-66e228ce5776e496afc63999c289b420019df0d20a3a717b9119259cf10e340fR112).  I'm down for changing both though!